charts/seed-bootstrap/charts/fluentd-es/templates/fluent-bit-daemonset.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: {{ include "daemonsetversion" . }}
+kind: DaemonSet
+metadata:
+  name: fluent-bit
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ toYaml .Values.fluentbit.labels | indent 4 }}
+spec:
+  selector:
+    matchLabels:
+{{ toYaml .Values.fluentbit.labels | indent 8 }}
+  template:
+    metadata:
```

Comment:
checksum annotation for the fluent-bit configmap is missing

Reply:
Done.